src/index.html

```diff
@@ -7,6 +7,308 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
```

Comment:
I recommend a variant of BEM with PascalCased “blocks”
```suggestion
    <header class="Header">
```